src/controller.rs

```diff
@@ -243,8 +244,12 @@ where
             .run(reconcile, error_policy, context)
             .for_each(|res| async move {
                 match res {
-                    Ok(o) => info!("Reconciled {:?}", o),
-                    Err(ref e) => error!("Reconcile failed: {:?}", e),
+                    Ok(o) => trace!(resource = ?o, "Reconciliation finished successfully (it is normal to see this message twice)"),
+                    Err(ref err) => {
+                        // If we get here it means that our `reconcile` method returned an error which should never happen because
+                        // we convert all errors to requeue operations.
+                        error!(?err, "Reconciliation finished with an error, this should not happen, please file an issue")
```

Comment:
Random brainfart, we are calling for people to file issues in some places, should we keep track of these places somewhere and give them an identifier that people can reference or something similar?
Nothing to do about this here, just a thought to be discussed.

Reply:
Yeah...we should reactivate our common logging thing but that's probably very much outdated by now. It's a good idea though.